### PR TITLE
2.13 release: tidy up & cut changelog

### PR DESCRIPTION
**Pull request by dimitarvdimitrov**
#### What this PR does

This cuts the changelog as of r295.

related to https://github.com/grafana/mimir/issues/8390

#### Checklist

- [ ] Tests updated.
- [ ] Documentation added.
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,33 +4,77 @@
...
-* [FEATURE] Alertmanager: Added `-alertmanager.max-silences-count` and `-alertmanager.max-silence-size-bytes` to set limits on per tenant silences. Disabled by default. #6898
+* [FEATURE] Alertmanager: Added `-alertmanager.max-silences-count` and `-alertmanager.max-silence-size-bytes` to set limits on per tenant silences. Disabled by default. #8241 #8249
 * [FEATURE] Ingester: add experimental support for the server-side circuit breakers when writing to and reading from ingesters. This can be enabled using `-ingester.push-circuit-breaker.enabled` and `-ingester.read-circuit-breaker.enabled` options. Further `-ingester.push-circuit-breaker.*` and `-ingester.read-circuit-breaker.*` options for configuring circuit-breaker are available. Added metrics `cortex_ingester_circuit_breaker_results_total`,  `cortex_ingester_circuit_breaker_transitions_total`, `cortex_ingester_circuit_breaker_current_state` and `cortex_ingester_circuit_breaker_request_timeouts_total`. #8180 #8285 #8315 #8446
```
Comment (dimitarvdimitrov) [inline CHANGELOG.md:78]: the last PR here still isn't backported to the 2.13 release, but will be once the branch is created, so I'm keeping the changelog entry in the 2.13 section

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,33 +4,77 @@
...
+### Query-tee
+
+* [ENHANCEMENT] Emit trace spans from query-tee. #8419
+* [ENHANCEMENT] Log trace ID (if present) with all log messages written while processing a request. #8419
+* [ENHANCEMENT] Log user agent when processing a request. #8419
+* [ENHANCEMENT] Add `time` parameter to proxied instant queries if it is not included in the incoming request. This is optional but enabled by default, and can be disabled with `-proxy.add-missing-time-parameter-to-instant-queries=false`. #8419
+* [BUGFIX] Ensure any errors encountered while forwarding a request to a backend (eg. DNS resolution failures) are logged. #8419
+
+### Documentation
+
```
Comment (dimitarvdimitrov) [inline CHANGELOG.md:42]:
```suggestion

```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,33 +4,76 @@
...
+* [ENHANCEMENT] Storage Provider: Added option `-<prefix>.s3.dualstack-enabled` that allows disabling S3 client from resolving AWS S3 endpoint into dual-stack IPv4/IPv6 endpoint. Defaults to true. #8405
+* [BUGFIX] Query-frontend: fix `-querier.max-query-lookback` enforcement when `-compactor.blocks-retention-period` is not set, and viceversa. #8388
+* [BUGFIX] Ingester: fix sporadic `not found` error causing an internal server error if label names are queried with matchers during head compaction. #8391
+* [BUGFIX] Ingester, store-gateway: fix case insensitive regular expressions not matching correctly some Unicode characters. #8391
+* [BUGFIX] Query-frontend: "query stats" log now includes the actual `status_code` when the request fails due to an error occurring in the query-frontend itself. #8407
+* [BUGFIX] Store-gateway: fixed a case where, on a quick subsequent restart, the previous lazy-loaded index header snapshot was overwritten by a partially loaded one. #8281
+
+### Mixin
+
+### Jsonnet
```
Comment (dimitarvdimitrov) [inline CHANGELOG.md:24]:
```suggestion
### Jsonnet
```

**Comment by pstibrany** [comment]
it's friday, there will be new weekly r296 on monday, perhaps we can create 2.13 from that branch instead?

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,33 +4,76 @@
 
 ### Grafana Mimir
 
+* [CHANGE] Store-gateway / querier: enable streaming chunks from store-gateways to queriers by default. #6646
+* [FEATURE] Querier: add experimental streaming PromQL engine, enabled with `-querier.query-engine=mimir`. #8348 #8430 #8454 #8455
```
Comment (pstibrany) [inline CHANGELOG.md:8]:
```
* [CHANGE] Querier: honor the start/end time range specified in the read hints when executing a remote read request. #8431
```

is missing from the changelog.

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,33 +4,76 @@
...
+* [ENHANCEMENT] Rules: Added per namespace max rule groups per tenant limit. The maximum number of rule groups per rule tenant for all namespaces continues to be configured by `-ruler.max-rule-groups-per-tenant`, but now, this can be superseded by the new `-ruler.max-rule-groups-per-tenant-by-namespace` option on a per namespace basis. This new limit can be overridden using the overrides mechanism to be applied per-tenant. #8425
+* [ENHANCEMENT] Query-frontend: be able to block remote read queries via the per tenant runtime override `blocked_queries`. #8372 #8415
+* [ENHANCEMENT] Query-frontend: added `remote_read` to `op` supported label values for the `cortex_query_frontend_queries_total` metric. #8412
+* [ENHANCEMENT] Query-frontend: log the overall length and start, end time offset from current time for remote read requests. The start and end times are calculated as the miminum and maximum times of the individual queries in the remote read request. #8404
+* [ENHANCEMENT] Storage Provider: Added option `-<prefix>.s3.dualstack-enabled` that allows disabling S3 client from resolving AWS S3 endpoint into dual-stack IPv4/IPv6 endpoint. Defaults to true. #8405
```
Comment (pstibrany) [inline CHANGELOG.md:15]: I'd suggest to keep this in 2.13 section, and let's backport this enhancement to the branch.

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,33 +4,76 @@
...
+* [BUGFIX] Query-frontend: fix `-querier.max-query-lookback` enforcement when `-compactor.blocks-retention-period` is not set, and viceversa. #8388
+* [BUGFIX] Ingester: fix sporadic `not found` error causing an internal server error if label names are queried with matchers during head compaction. #8391
+* [BUGFIX] Ingester, store-gateway: fix case insensitive regular expressions not matching correctly some Unicode characters. #8391
+* [BUGFIX] Query-frontend: "query stats" log now includes the actual `status_code` when the request fails due to an error occurring in the query-frontend itself. #8407
+* [BUGFIX] Store-gateway: fixed a case where, on a quick subsequent restart, the previous lazy-loaded index header snapshot was overwritten by a partially loaded one. #8281
```
Comment (pstibrany) [inline CHANGELOG.md:20]: We need to add changelog entry from #8462 here (should be fixed by rebase).

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,33 +4,76 @@
...
+
+### Mixin
+
+### Jsonnet
+
+* [ENHANCEMENT] Update rollout-operator to `v0.17.0`. #8399
```
Comment (pstibrany) [inline CHANGELOG.md:26]: We want to release this with 2.13.

**Review by pstibrany — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (dimitarvdimitrov) [reply CHANGELOG.md:8]:
```suggestion
* [FEATURE] Querier: add experimental streaming PromQL engine, enabled with `-querier.query-engine=mimir`. #8422 #8430 #8454 #8455
```

**Pull request merged**